package.json

```diff
@@ -184,24 +184,18 @@
 		"@testing-library/jest-dom": "^5.16.5",
 		"@testing-library/react": "^10.4.9",
 		"@testing-library/react-hooks": "^3.3.0",
-		"@wordpress/babel-preset-default": "^4.17.0",
```

Comment:
Looks like removing this dependency caused issues in some of the workflows because they somehow depend on it. Probably because we use `@wordpress/default` in the babel config 🤔. Let's revert this and add this dependency in the main package.json as well.